crate.go

```diff
@@ -113,8 +113,8 @@ func newCrateEndpoint(ep *endpointConfig) *crateEndpoint {
 
 func (c *crateEndpoint) endpoint() endpoint.Endpoint {
 	/**
-	 * Initialize connection pools lazily here instead of in `newCrateEndpoint()`,
-	 * so that the adapter does not crash on startup if the endpoint is unavailable.
+	* Initialize connection pools lazily here instead of in `newCrateEndpoint()`,
```

Comment:
super minor: I think the indentation was correct before here?

Reply:
I believe indentation is handled automatically by **gofmt**. However, as far as I know, Javadoc-style comments aren't commonly used in **Go**, though my knowledge on this is somewhat limited.

Reply:
changed it to follow go comment guidelines

Reply:
thx!